### DISCUSSION TITLE
Documentation issues - line length  and initrd

### DISCUSSION THREAD

**Discussion by JohnLar42** [ipxe/ipxe#273] [Ideas]
After much failure discovered the iPXE documentation is missing 2 important aspects.
1. Line length is limited, you can't put more than around 256 characters or so of arguments into a single chain command. You can get around this by setting variables and including these variables in the chain command.  
  set arg1 = blah blah  
  chain vmlinuz ${arg1}

2. You need to specify initrd=initrd.img generally. Documentation implied that you don't (maybe this worked in older versions??)
  kernel \kernels\vmlinuz initrd=initrd.img  {$args}
  initrd \kernels\initrd.img
  boot
or
 kernels \kernels\vmlinuz
 initrd \kernels\initrd.img
 chain vmlinuz initrd=initrd.img  {$args}

both work, while documentation often leaves the initrd=initrd.img off which is generally required.

**Comment by NiKiZe** [comment]
The requirement for specifying the initrd to the kernel in efi mode is well documented, but is not really ipxe specific but rather efikernel specific.
